### ISSUE TITLE
is this still needed ?

### ISSUE THREAD

**Issue by triatomic** [HW-OLE/ax3200-ax6s-mtk-openwrt#1]
Doesn't Upstream openwrt have mtk wifi drivers baked in already ?

**Comment by HW-OLE** [comment]
No, it uses the open source mt76 driver but not the proprietary, non open-source drivers from mediatek

**Comment by triatomic** [comment]
how can i be sure my compiled image is using the non open source drivers ?
i followed the steps in your repo obviously.

**Comment by HW-OLE** [comment]
The compilation seems to have never baked the MTK drivers in. I'm working with a few people that wanted to gave me instructions some time ago but I will definitely keep this repo updated if I can finally build an image
